docassemble/ALKilnTests/data/sources/observation_steps.feature

```diff
@@ -99,23 +99,26 @@ Scenario: Test "Then I don’t continue" with an apostrophe
   And I will be told an answer is invalid
 
 @fast @o9 @table @json
-Scenario: I can match JSON page var to str
-  Given I start the interview at "all_tests.yml"
-  And I get to "showifs" with this data:
-    | var | value | trigger |
-    | double_quote_dict["double_quote_key"]['dq_two'] | true |  |
-    | checkboxes_other['checkbox_other_opt_1'] | true |  |
-    | dropdown_test | dropdown_opt_2 | |
-    | radio_yesno | False | false |
-    | radio_other | radio_other_opt_3 | |
-    | single_quote_dict['single_quote_key']['sq_two'] | true |  |
-    | text_input | Regular text input field value | |
-    | textarea | Multiline text\narea value | |
-    | date_input | today | |
-  Then the text in the JSON variable "dropdown_test" should be
+Scenario: I can match JSON page vars
+  Given I start the interview at "test_json.yml"
+  Then the text in the JSON variable "multiline_val" should be
+  """
+  This string contains.
+  A new line character.
+  """
+  Then the text in the JSON variable "multiline_block" should be
+  """
+  This block is.
+  A multiline block.
+  """
+  Then the text in the JSON variable "single_line_val" should be
   """
-  dropdown_opt_2
+  This is one line.
   """
+  #Then the text in the JSON variable "number_val" should be
```

Comment:
From the comments in the issue, it looks like checking the number wasn't working properly. I'd remove the commented lines here and the `number_val` variable from the yaml interview if we don't want to try to make it work in this PR.